src/mmds/src/data_store.rs

```diff
@@ -89,15 +105,7 @@ impl Mmds {
         self.data_store.to_string()
     }
 
-    /// This function replicates the behavior of the Instance Metadata Service
```

Comment:
Can we move this comment to the below function instead of removing it?

Reply:
This comment is not relevant anymore. The method returns a plain JSON now.